opentreemap/treemap/instance.py

```diff
@@ -149,7 +158,10 @@ class Instance(models.Model):
         'BenefitCurrencyConversion', null=True, blank=True)
 
     """ Center of the map when loading the instance """
-    bounds = models.MultiPolygonField(srid=3857)
+    bounds = models.MultiPolygonField(srid=3857, null=True, blank=True)
```

Comment:
Based on what we discussed as the plan for making this safe for a blue/green deployment, I think we need to *remove* this field (without checking in the accompanying migrations)